frodo.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?